### DISCUSSION TITLE
Does Frigate not support multiple live views of many cameras at a time?

### DISCUSSION THREAD

**Discussion by mike6789** [blakeblackshear/frigate#13818] [Ask A Question]
I am noticing on the tile view, only one camera is live (with the red dot indicator). The other shows a stale frame... Is this how Frigate is designed? I want to see multiple cameras tiled all updating the live view.

**Comment by NickM-27** [comment]
any camera that is detecting motion or active objects will be live. There is no point to stream a camera that does not have motion because it would be the same as a live view except using more resources and bandwidth
